### PR TITLE
Working on toggles

### PR DESCRIPTION
**Pull request by Ar3Tea**
This PR is adding: 
* the toggle for expanding/collapsing all movies.
* the popup for all info for each movie

Reworking of the API component will come after

### CODE REVIEW

File: movie-browser/src/App.js [hunk 1]
```diff
@@ -1,18 +1,30 @@
-import React from 'react';
+import React, { Component } from 'react';
 import Header from './components/Header.js';
 import Featured from './components/Featured.js';
 import Movies from './components/Movies.js';
 import Footer from './components/Footer.js';
+//import fetchMovies from './components/Api.js';
```
Comment (jpggvilaca) [inline movie-browser/src/App.js:6]: delete comments. Same for line 16

File: movie-browser/src/components/Featured.js [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
-            movies: []
-        };
-    }
-  
     componentDidMount() {
       fetch('https://ghibliapi.herokuapp.com/films')
-        .then(response => response.json())
-        .then(movies => this.setState({ movies }));
+      .then(response => response.json())
+      .then(data => this.setState({ movies: data }));
```
Comment (jpggvilaca) [inline movie-browser/src/components/Featured.js:10]: you should use your api's `fetchMovies` here

File: movie-browser/src/components/Movies.js [hunk 3]
```diff
@@ -1,36 +1,56 @@
...
-    constructor(props) {
-        super(props);
+        state = {
+            movies: [],
+            shouldShowAllMovies: false
+        };
 
-        this.state = {
-            movies: []
+        componentDidMount() {
```
Comment (jpggvilaca) [inline movie-browser/src/components/Movies.js:11]: you should use your api's `fetchMovies` here

File: movie-browser/src/index.css [hunk 4]
```diff
@@ -1,10 +1,13 @@
-@tailwind base;
-
-@tailwind components;
-
-@tailwind utilities;
+.footer,
```
Comment (jpggvilaca) [inline movie-browser/src/index.css:1]: are you still using `tailwind`? If not..remove it from the dependencies.

File: movie-browser/src/App.js (hunk 1 above)
Comment (Ar3Tea) [reply movie-browser/src/App.js:6]: I left them in there for the future use of api.js but wanted the app to work before refactoring. I can remove if you still think it should be.

File: movie-browser/src/components/Featured.js (hunk 2 above)
Comment (Ar3Tea) [reply movie-browser/src/components/Featured.js:10]: I will refactor into using the API.js component. I wanted the app to work first.

File: movie-browser/src/components/Movies.js (hunk 3 above)
Comment (Ar3Tea) [reply movie-browser/src/components/Movies.js:11]: I will refactor into using the API.js component. I wanted the app to work first.

File: movie-browser/src/index.css (hunk 4 above)
Comment (Ar3Tea) [reply movie-browser/src/index.css:1]: Yes. Tailwind is the CSS framework in use alongside custom CSS. I've heard good things about it and figured this project was a good time to try it out.

File: movie-browser/src/App.js (hunk 1 above)
Comment (jpggvilaca) [reply movie-browser/src/App.js:6]: oh, that's ok then. just leave them for now.

File: movie-browser/src/components/Featured.js (hunk 2 above)
Comment (jpggvilaca) [reply movie-browser/src/components/Featured.js:10]: alright

File: movie-browser/src/index.css (hunk 4 above)
Comment (jpggvilaca) [reply movie-browser/src/index.css:1]: it's just that i saw u remove the imports. That's why i asked. But if you're still using it that's perfectly fine

**Review by jpggvilaca — APPROVED** [review]

**Pull request merged**